Source/Renderer/CubeMap.js

```diff
@@ -313,6 +321,9 @@ define([
         gl.bindTexture(target, this._texture);
         gl.generateMipmap(target);
         gl.bindTexture(target, null);
+
+        // The mipmap adds approximately 1/3 of the original texture size
+        this._sizeInBytes = Math.floor(this._sizeInBytes * 4 / 3);
```

Comment:
A user could call `generateMipmap` more than once.  Either save the original size and recompute here, or just compute every time in the `get` function based on if this is mipmapped.